config/application.yml.tt

```diff
@@ -13,3 +13,11 @@ development:
 test:
   <<: *default
   TEST_RETRY: "0"
+
+# Set environment variables required in the initializers in order to precompile the assets.
+# Because it initializes the app, so all variables need to exist in the build stage.
+production:
```

Comment:
To avoid the confusion between this and the real Production, how about we rename this one to `build` 🤔 That means some env during the `build` step 🤔 

or `compile` or `pre_production` 🌩️